doc/api/buffer.md

```diff
@@ -1819,8 +1819,8 @@ console.log(buf.readUInt32LE(1).toString(16));
 added: v0.11.15
 -->
 
-* `offset` {integer} Where to start reading. Must satisfy: `0 <= offset <= buf.length - byteLength`
-* `byteLength` {integer} How many bytes to read. Must satisfy: `0 < byteLength <= 6`
+* `offset` {integer} Where to start reading. Must satisfy: `0 <= offset <= buf.length - byteLength`.
```

Comment:
Also not related to this PR, but "where to start reading" and "how many bytes to read" could probably be rewritten to sound less like questions.